docs/CUSTOMIZING.md

```diff
@@ -0,0 +1,66 @@
+# Customizing guide
```

Comment:
I would go with "Customization Guide" here as well.

Reply:
:heavy_check_mark: Changed in f49d549452dbe6b2e10b8050ace391d0ee12b55f.